src/KubeOps/Operator/Builder/OperatorBuilder.cs

```diff
@@ -84,6 +95,13 @@ public IOperatorBuilder AddFinalizer<TFinalizer>()
             return this;
         }
 
+        public IOperatorBuilder AddResourceAssembly(Assembly assembly)
```

Comment:
this call must be added to the `IOperatorBuilder` interface, otherwise it won't be accessible.

Reply:
Fixed. Also added method documentation.